examples/image/compute_gray_with_squidpy.im.ImageContainer.apply

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python
+"""
+Convert to grayscale using squidpy.im.ImageContainer.apply
+--------------------
+This example shows how to use :func:`squidpy.ImageContainer.apply` to convert an image layer to grayscale.
+
+We take the mean of all the three channels (RGB) using the defined function `rgb2gray` to convert the image to grayscale.
+The function `rgb2gray` calls numpy method mean to calculate the mean (average) of three channels (RGB) across axis 4.
+
+.. seealso::
+    - :ref:`sphx_glr_auto_examples_image_compute_smooth.py`
+    - :ref:`sphx_glr_auto_examples_image_compute_process_hires.py`
+
```

Comment:
Did you run pre-commit before? I don't think this space should be here and pre-commit would have catched it